Develop

YulianaGusakova opened the pull request.

YulianaGusakova committed:
Создан CourierCreateTest

YulianaGusakova committed:
Создан CourierLoginTest

YulianaGusakova committed:
Создан MakeOrderColorTest и GetOrderListTest, добавлены аннотации @Step

YulianaGusakova committed:
Создан GetOrderListTest, изменения в pom.xml

YulianaGusakova committed:
add allure report

YulianaGusakova committed:
Внесены изменения в pom.xml

YulianaGusakova committed:
Update README.md

YulianaGusakova committed:
Update README.md

YulianaGusakova committed:
Update README.md

okryukova commented on src/main/java/POJO/Courier.java:
```diff
@@ -0,0 +1,44 @@
+package POJO;
```
⛔️Нужно исправить. Необходимо исправить наименование пакетов в соответствии с правилами нейминга. Пакеты — допустимы только строчные буквы. В случае, если слов несколько, их следует разделять точкой. Префикс должен быть одним из доменных имен верхнего уровня, таких как com, ru и т.д. Наименование пакета должно отражать суть содержимого. https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

okryukova commented on src/main/java/POJO/Courier.java:
```diff
@@ -0,0 +1,44 @@
+package POJO;
+
+public class Courier {
```
⚠️Можно улучшить. Можно использовать @DaTa и @AllArgsConstructor из библиотеки Lombok

okryukova commented on src/test/java/CourierLoginTest.java:
```diff
@@ -0,0 +1,139 @@
+    @DisplayName("Успешная авторизация при передаче всех обязательных полей")
+
+    public void courierLoginTest() {
+        courierSteps
```
⛔️Нужно исправить. Если для теста нужен пользователь, то он создается в методе с аннотацией [@Before](http://staff.yandex-team.ru/Before).

okryukova commented on src/test/java/CourierLoginTest.java:
```diff
@@ -0,0 +1,139 @@
+
+    @Test
+    @DisplayName("Авторизация без логина")
+
```
⚠️Можно улучшить. В allure есть еще одна аннотация для описания теста Description. Добавь ее, пожалуйста, во всех тестах проекта

YulianaGusakova committed:
1.  Исправлено наименование пакетов в соответствии с правилами нейминга
2. Создание курьера вынесено из метода courierLoginTest() в метод setUP() с аннотацией @Before
3. Создание заказа из метода orderList() вынесено в метод setUpGetOrderListTest() с аннотацией @Before
4. Все тесты проекта помечены аннотацией @Description
5. Обновлен отчет Allure

YulianaGusakova committed:
Merge remote-tracking branch 'origin/develop' into develop

YulianaGusakova committed:
Update README.md

YulianaGusakova committed:
Update README.md

YulianaGusakova committed:
Update README.md
